### DISCUSSION TITLE
Debounce table individual search column

### DISCUSSION THREAD

**Discussion by clinvest** [filamentphp/filament#9654] [Help]
Hi all,

Is there a way to debounce the individual search column? sometimes it seaches a bit fast and I don't have time to type.

`TextColumn::make('name')->label(__('admin.place'))->searchable(isIndividual: true, isGlobal: false)`

**Comment by leandrocfe** [comment]
debounce is static `:wire:model.live.debounce.500ms="$wireModel"`. You could create a custom view in `resources/views/vendor/filament-tables/components/search-field.blade.php`, copy the original content `vendor/filament/tables/resources/views/components/search-field.blade.php` and edit what you want. But [have a look](https://filamentphp.com/community/danharrin-publishing-views-in-laravel).

You could also send a PR.

**Comment by mackhankins** [comment]
It's way to fast.  Never thought I'd be saying that.
